platforms/fabric/src/main/java/com/dfsek/terra/fabric/mixin/lifecycle/ChunkGenerator_ChunkGeneratorRegistrationMixin.java

```diff
@@ -0,0 +1,19 @@
+package com.dfsek.terra.fabric.mixin.lifecycle;
+
+
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import com.dfsek.terra.fabric.FabricEntryPoint;
+
+
+@Mixin(ChunkGenerator.class)
+public class ChunkGenerator_ChunkGeneratorRegistrationMixin {
```

Comment:
Underscore in name?

`ChunkGeneratorRegistrationMixin` should be fine, no?

Reply:
The underscore specifies what specifically this mixin does.